modules/nw-multus-configure-dualstack-ip-address.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_content-type: PROCEDURE 
+
+[id="nw-multus-configure-dualstack-ip-address_{context}"]
+= Creating a configuration for assignment of dual-stack IP addresses dynamically
+
+Dual-stack IP address assignment can be configured with the `ipRanges` parameter for:
+* IPv4 addresses
+* IPv6 addresses
+* multiple IP address assignment 
+
+.Procedure
+
+. Set type to whereabouts.
+. Use `ipRanges` to allocate IP addresses.
+
+[source,yaml]
+----
+cniVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  additionalNetworks:
+  - name: whereabouts-shim
+    namespace: default
```

Comment:
Need add `type: Raw`

Reply:
done

Reply:
@weliang1 can you confirm this change is correctly implemented?

Reply:
@nmuller66 One more thing.
Could you update `{"range": "192.168.10.1/24"},` to `{"range": "192.168.10.0/24"},`?

Reply:
@weliang1 absolutely yes

Reply:
@weliang1 with this update, is that a LGTM from a QE perspective